### PR TITLE
[MSHARED-787] Add optional buildEnvironment information to the manifest

### PR DESCRIPTION
**Pull request by michael-o**

### CODE REVIEW

File: src/main/java/org/apache/maven/archiver/MavenArchiver.java [hunk 1]
```diff
@@ -615,6 +620,9 @@ public void createArchive( MavenSession session, MavenProject project,
         Manifest manifest = getManifest( session, workingProject, archiveConfiguration );
 
         // Configure the jar
+
+        archiver.setMinimalDefaultManifest( true );
```
Comment (eolivelli) [inline src/main/java/org/apache/maven/archiver/MavenArchiver.java]: This change  seems unrelated

Comment (michael-o) [reply src/main/java/org/apache/maven/archiver/MavenArchiver.java]: Thanks for pointing that out. This belongs into another ticket. I will drop this hunk.

Comment (eolivelli) [reply src/main/java/org/apache/maven/archiver/MavenArchiver.java]: you are welcome !
cheers

**Comment by jira-importer** [comment]
Resolve #217
